containers/release.Containerfile

```diff
@@ -0,0 +1,86 @@
+FROM quay.io/centos/centos:stream9 as builder
```

Comment:
Shouldn't we use: Base image UBI 9?

Reply:
Yeah, we can use UBI 9 for the builder stage! Updated